tidy3d/components/base.py

```diff
@@ -563,7 +563,7 @@ def to_hdf5(self, fname: str, custom_encoders: List[Callable] = None) -> None:
 
         with h5py.File(fname, "w") as f_handle:
 
-            f_handle[JSON_TAG] = self._json_string
+            f_handle[JSON_TAG] = self._json()
```

Comment:
does this `indent=None` by default? maybe it's more clear to pass it explicitly?